test/mockdata/mockSoknader.ts

```diff
@@ -465,7 +465,7 @@ const mockSoknader = [
         tag: "VAER_KLAR_OVER_AT",
         undersporsmal: [],
         undertekst:
-          "<ul><li>rett til sykepenger forutsetter at du er borte fra arbeid på grunn av egen sykdom. Sosiale eller økonomiske problemer gir ikke rett til sykepenger</li><li>du kan miste retten til sykepenger hvis du uten rimelig grunn nekter å opplyse om egen funksjonsevne eller nekter å ta imot tilbud om behandling og/eller tilrettelegging</li><li>sykepenger utbetales i maksimum 52 uker, også for gradert (delvis) sykmelding</li><li>fristen for å søke sykepenger er som hovedregel 3 måneder</li></ul>",
+          "Dette er tekst som kun vises til sykmeldt om ting de må være klar over ved innsending av søknad",
```

Comment:
Endret til en sånn tekst, mest som en reminder. Men jeg vet ikke om dere heller vil at vi beholder den gamle teksten? Den gamle teksten er ikke lenger det som vises i frontend hos flex, den er utdatert den og.

Reply:
Mulig jeg ikke forstår helt, men kan vi ikke her bare fjerne hele (denne delen av) objektet? Siden vi ikke skal støtte `VAER_KLAR_OVER_AT`

Reply:
Tja, jeg tenkte at hvis flex fortsatt sender denne infoen, så burde mocken reflektere at her kommer det noe data. Men jeg venter egentlig litt på svar om de faktisk sender dette fortsatt eller ikke, Nils Jørgen skulle sjekke det opp, men tror han har glemt det.

Reply:
Aha okei. Hvis de fortsatt skal sende det så bør vi jo beholde den høres det ut som. Hvis ikke er det kanskje bare å fjerne hele `VAER_KLAR_OVER_AT` opplegget?

Reply:
Ser ut som vi kan fjerne alt https://nav-it.slack.com/archives/CMF67JLRM/p1751012439993459?thread_ts=1750260276.373679&cid=CMF67JLRM

Reply:
Yihu! 🙌